nixos/modules/services/monitoring/netdata.nix

```diff
@@ -140,7 +151,9 @@ in {
       path = (with pkgs; [ curl gawk which ]) ++ lib.optional cfg.python.enable
         (pkgs.python3.withPackages cfg.python.extraPackages);
       serviceConfig = {
-        Environment="PYTHONPATH=${cfg.package}/libexec/netdata/python.d/python_modules";
+        Environment = [
```

Comment:
I think it'd be cleaner to use `systemd.services.netdata.environment` and the nix style syntax for specifying the environment.